projects/ngx-quill/src/lib/quill.service.ts

```diff
@@ -1,18 +1,61 @@
 import { DOCUMENT } from '@angular/common'
 import { Injectable, Inject, Injector, Optional } from '@angular/core'
+import { defer, Observable, shareReplay } from 'rxjs'
```

Comment:
@KillerCodeMonkey 

This import introduced breaking change for apps using `rxjs@6`. 

Since angular hasn't yet bumped rxjs would be good to bring back `rxjs@6` support. 
It can be achieved by using `import { shareReplay } from 'rxjs/operators'` instead.

Reply:
dang it. i missed that. -.-

Reply:
@kamilchlebek check 16.1.1

Reply:
Works perfectly, thank you! 🚀